src/settings.cpp

```diff
@@ -0,0 +1,13 @@
+#include "settings.hpp"
+
+settings::settings(int motorSpeed) : motorSpeed(motorSpeed) {
+}
+
+void settings::setMotorSpeed(int speed) {
+    motorSpeed = speed;
+    hwlib::cout << "Set motorspeed to: " << motorSpeed << hwlib::endl;
```

Comment:
also explain in the doxygen that for right now it will just print the speed and that the functionality will be added in a later sprint.